### DISCUSSION TITLE
Arch build

### DISCUSSION THREAD

**Discussion by DeltaCopy** [Bali10050/Darkly#12] [Show and tell]
I've created a new PKGBUILD which allows you to build and install Lightly on Arch.

https://github.com/DeltaCopy/lightly-archlinux

Plan to move to AUR once the package name is finalized.

**Comment by RafaelJust** [comment]
Oh, I've already made one two days ago at https://aur.archlinux.org/packages/lightly-fork-git

But since I'm pretty new at making PKGBUILDs, it would be beter to check if yours works beter, and replace mine :)

**Reply by DeltaCopy** [reply]
Nice. The only things I see different are.

- validating the src integrity using the sha256sum
- use the release tag as the src rather directly from the branch
- have no reference to QT5 dependencies

**Reply by RafaelJust** [reply]
Yeah I saw you PKGBUILD and it's better than mine -_-
How can I transfer ownership of the AUR package? (sorry I don't know a lot about maintaing AUR packages)

**Reply by DeltaCopy** [reply]
No idea unfortunately, i've never used the AUR except for installing from it.

**Reply by RafaelJust** [reply]
OK I foubd a way: can you give me your AUR username?
I can add you as a maintainer then.

**Reply by DeltaCopy** [reply]
Same as here. DeltaCopy

**Reply by RafaelJust** [reply]
I added you to the maintainers list! You can replace it with yours now :)
I will probably disown the package so you become the main maintainer.

**Reply by DeltaCopy** [reply]
I didn't want to obliterate the work you've made already.
So I created my own package called lightly-qt6.
There were a few additional tweaks I made to the PKGBUILD around how make is utilizing the available CPU cores.
